tests/test_edgeql_ddl.py

```diff
@@ -6445,9 +6445,14 @@ async def test_edgeql_ddl_extending_04(self):
                     (SELECT ggc.properties FILTER .name = 'a')
                         .annotations@value;
             """,
-            {}
+            []
         )
 
+    @test.xfail('''
+        Default value ought to get reset back to non-existent, since it
+        was inherited? (Or actually maybe not, since the prop is owned
+        by then?)
```

Comment:
This test is the one questionable one. It wanted to test that when a parent type is dropped, a owned inherited property that was inheriting the default should lose the default.

It wasn't quite testing this, because it was dropping one parent a second time (#3198).

When that is fixed, though, dropping the remaining parent doesn't cause the default to clear.

Maybe that is right, though? The child property is owned. (Though the default is inherited.)

Reply:
The answer to such DDL questions is: how would you express an equivalent SDL migration.  Losing a parent in SDL usually means you lose everything you inherited from it, so IMO it's natural that DDL follows the behavior.